test/IECoreHoudini/SceneCacheTest.py

```diff
@@ -2319,8 +2319,9 @@ def testRopTopLevelGeo( self ) :
 
 		rop.parm( "execute" ).pressButton()
 
-		# Skips over 3 cortex objects.
-		self.assertEqual( len( rop.errors() ) , 0 )
+		# unable to convert the 3 cortex objects.
+		# "Unable to convert SOP: '/obj/ieSceneCacheGeometry1' prim types found: [ CortexObject ]"
```

Comment:
i get it, but this seems like an odd thing to be unable to convert. the error looks like we cant write cortex objects into an scc (whereas i guess the real error is that its an ObjectVector and we dont have a converter for that?). i guess in practice this will come up more often as 'prim types found [ polygon nurbs ]' or something like that?

Reply:
yup. It's for the multiple prim types in a SOP case.

Reply:
did you run the language by carsten? does he think its clear enough?

Reply:
updated in 1ced5cf based on Carsten's feedback.